test/e2e/bluegreen_test.go

```diff
@@ -158,3 +158,121 @@ spec:
 		Then().
 		ExpectRevisionPods("revision 2 has active metadata2", "2", podsHaveActiveMetadata2)
 }
+
+func (s *BlueGreenSuite) TestBlueGreenProgressDeadlineExceededWithPause() {
+	s.Given().
+		RolloutObjects(`
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollout-bluegreen
+spec:
+  replicas: 2
+  revisionHistoryLimit: 2
+  progressDeadlineSeconds: 10 # note this is less than initialDelaySeconds
+  selector:
+    matchLabels:
+      app: rollout-bluegreen
+  template:
+    metadata:
+      labels:
+        app: rollout-bluegreen
+    spec:
+      containers:
+      - name: rollouts-demo
+        image: argoproj/rollouts-demo:blue
```

Comment:
Could we use the nginx image instead? All the other e2e tests use it and it helps with keeping the e2e tests running quicker since it does not need to pull images.

Reply:
Is it possible for me to use a readiness probe when using the nginx image? I need to configure a readiness probe to set `initialDelaySeconds`.

Reply:
Yes it just need to be modified to the port which nginx listens on instead of rollouts-demo